frontend/src/components/UASTViewer.js

```diff
@@ -40,12 +37,27 @@ class UASTViewer extends Component {
     api
       .filterUAST(this.props.protobufs, this.state.filter)
       .then(uast => {
-        this.setState({ uast: transformer(uast) });
+        this.setState({ uast: this.transform(uast) });
       })
       .catch(err => this.setState({ uast: null, error: err }))
       .then(() => this.setState({ loading: false }));
   }
 
+  // Applies the uast-viewer object shape transformer, and expands all nodes
```

Comment:
before it was expanding only first 2 levels.
The problem with expanding everything - uast is huge. For a big file, it will hang a browser.

Reply:
there is [expandRootIds](https://github.com/bblfsh/uast-viewer/blob/master/src/helpers.js#L105) helper for that. Before it was included in `transformer` but caused some problems so now you need to call it explicitly.

Reply:
done in a8cacfd